<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="MIKEY-MAC-095" failures="1" tests="5" name="Personal loan" time="0.049" errors="0" timestamp="2023-04-30T22:05:31 EDT">
  <testcase classname="Test.Day1" name="priyank2" time="0.007"/>
  <testcase classname="Test.day2" name="test2" time="0.003"/>
  <testcase classname="Test.day3" name="MobileLogincarLoan" time="0.001"/>
  <testcase classname="Test.day3" name="WeblogincarLoan" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Test.day3.WeblogincarLoan(day3.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- WeblogincarLoan -->
  <testcase classname="Test.day3" name="LoginAPIcarLoan" time="0.002"/>
  <testcase classname="Test.Day1" name="test2" time="0.0">
    <ignored/>
  </testcase> <!-- test2 -->
  <testcase classname="Test.Day1" name="priyank" time="0.0">
    <ignored/>
  </testcase> <!-- priyank -->
</testsuite> <!-- Personal loan -->
